Cargo.toml

```diff
@@ -1,21 +1,23 @@
 [package]
-name = "sr25519crust"
-version = "0.1.0"
-authors = ["Bogdan Vaneev <warchantua@gmail.com>"]
+name = "schnorrkel_crust"
+version = "0.2.0"
+authors = ["Bogdan Vaneev <warchantua@gmail.com>", "Vladimir Shcherba <abrehchs@gmail.com>"]
 edition = "2018"
```

Comment:
```suggestion
edition = "2020"
```

Reply:
Rust editions are released every 2-3 years. There are 2015 and 2018. Seems like 2021 will be the next one.

Reply:
Yep! And wrong edition usage causes very interesting compilation bugs.